### PR TITLE
Issue #15218: formatted input file for section 4.4 Column Limit to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java [hunk 1]
```diff
@@ -1,236 +1,243 @@
...
+  // href="www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search
+  // violation above 'Line is longer than 100 characters (found 111).'
+  int invalidHref = 88;
+
+  // Very long url with valid href:
+  // href="www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
+  int validHref = 54;
+
+  // Very long url with valid href: href    = "www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
+  int validHrefWithWhiteSpaces = 54;
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java:221]: FYI: Formatter formatted this line to something like:

```java
  // Very long url with valid href: href    =
  // "www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
```

but this line was essential to test the section properly so I reverted formatter's changes

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java [hunk 2]
```diff
@@ -1,236 +1,243 @@
...
+  // ftp://ftp.example.com/areallyyyyyyyyyyyylongggggggggggggggggggggggurlllll.text
+  int ftp = 0;
+
+  // Very long url with invalid href:
+  // href="www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search
+  // violation above 'Line is longer than 100 characters (found 111).'
+  int invalidHref = 88;
+
+  // Very long url with valid href:
+  // href="www.google.com/search?hl=en&q=java+style+guide+checkstyle+check+href+length+limit&btnG=Google+Search"
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java:217]: please restore this lines only.
all other should stay formatted.
in this chapter  we care about long urls in complicated cases.
We will not do auto formatting on this file.

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java (hunk 2 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java]: done

**Comment by romani** [comment]
please fix CI:
```
[ERROR]   ColumnLimit100Test.testLineLength:36->AbstractGoogleModuleTestSupport.verifyWithConfigParser:186
->AbstractItModuleTestSupport.verifyWithItConfig:471
->AbstractItModuleTestSupport.verifyViolations:549 Actual and expected violations differ.

expected to match: 209:(?:\d+:)?\s.*Line is longer than 100 characters \(found 111\)..*
but was          : 209: Line is longer than 100 characters (found 144).
[INFO] 
[ERROR] Tests run: 610, Failures: 1, Errors: 0, Skipped: 0
```

**Review by romani — APPROVED** [review]
ok to merge if CI is green

**Comment by rdiachenko** [comment]
@Zopsss please check and address ci failures

**Review by romani — APPROVED** [review]

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**